.azure-pipelines/unit_tests.yml

```diff
@@ -48,3 +48,11 @@ jobs:
     displayName: Install tox
   - script: tox
     displayName: Run unit tests
+  - script: tox -e integration -- --vcr-record=none
+    displayName: Run integration tests on cassettes
+    env:
+      DD_TEST_CLIENT_USER: frog@datadoghq.com
```

Comment:
Let's store that in a variable in azure

Reply:
the problem is that it must be a constant that was used for recording

Reply:
Not sure why that's a problem

Reply:
If I don't keep it here, then I should store it together with cassettes similar to `seed` or `frozen` time.

Reply:
I am storing the `user_handle` per test case together with cassettes.